.github/workflows/dependency-updates-post-processing.yaml

```diff
@@ -9,6 +9,7 @@ on:
 jobs:
   dependency-updates-post-processing:
     name: "Dependency Updates Post-Processing"
+    if: "!contains(github.event.pull_request.labels.*.name, 'skip-dependency-postprocessing')"
```

Comment:
This label is pretty self-explanatory - it was needed for this PR as sometimes the  automated postprocessing fails and actually rolls back the changes.